### PR TITLE
[plugin-web-app] Use new headless mode

### PR DESCRIPTION
**Pull request by valfirst**
See details: https://github.com/SeleniumHQ/selenium/pull/11099#issue-1402374967

### CODE REVIEW

File: vividus-plugin-web-app/src/main/resources/properties/profile/web/headless/chrome/profile.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 selenium.browser=chrome
-web.driver.CHROME.command-line-arguments=--headless --hide-scrollbars
+web.driver.CHROME.command-line-arguments=--headless=chrome --hide-scrollbars
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/resources/properties/profile/web/headless/chrome/profile.properties:2]: can it be a potential breaking change for end users? as per https://chromium.googlesource.com/chromium/src/+/833543a511a970f44fb074b9f30c6b6268e0be71 its completely different browser implementation

Comment (valfirst) [reply vividus-plugin-web-app/src/main/resources/properties/profile/web/headless/chrome/profile.properties:2]: see initial PR comment

**Comment by valfirst** [comment]
Chrome dev team is still playing with options (https://www.selenium.dev/blog/2023/headless-is-going-away/#what-are-the-two-headless-modes):
> The traditional --headless, and since version 96, Chrome has a new headless mode that allows users to get the full browser functionality (even run extensions). Between versions 96 to 108 it was --headless=chrome, after version 109 --headless=new.

**Comment by valfirst** [comment]
And more details:
https://developer.chrome.com/articles/new-headless/: 
> Currently, passing the `--headless` command-line flag without an explicit value still activates the old Headless mode — but we plan to change this default to new Headless over time.

Unofficial ETA: 2023Q1 - 2023Q2

**Pull request closed without merging**